usr/lib/systemd/system/apisix.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=apisix
+Conflicts=apisix.service
+After=network-online.target
+
+[Service]
+Type=forking
+WorkingDirectory=/usr/bin
```

Maybe we can use the same PWD with the Nginx process?
```suggestion
WorkingDirectory=/usr/local/apisix
```

---

Thanks for suggestion!